src/components/dashboard-card/ASDashboardCard.tsx

```diff
@@ -0,0 +1,53 @@
+import { View, Text, Image } from 'react-native'
+import React from 'react'
+
+import { DASHBOARD_LIST_DATA } from '../../constants/dashboard-constants'
+import ASPlayButton from '../play-button/ASPlayButton'
+import { IDashboardData } from '../../types/dashboard-types'
+import { styles } from './asDashboardCard-styles'
+
+const PLAY_BUTTON = require('../../assets/icons/audio.png')
+const COMPLETED_TICK = require('../../assets/icons/complete.png')
+const BOOKMARK = require('../../assets/icons/marked-favourite.png')
+const REMOVE_BOOKMARK = require('../../assets/icons/not-marked-favourite.png')
+
+interface IASDashboardCardProps {
```

Comment:
use the interface which you already made in types dashboard-types

Reply:
I think its correct this way since we have a data prop inside the props object which implements IDashboardData.